CNV-56896: Virt and hosted control plane confusion (#108122)
openshift/openshift-docs: modules/virt-aws-bm.adoc

* You can use any storage solution that is certified by the storage vendor to work with the underlying platform.
endif::openshift-dedicated[]
ifdef::openshift-dedicated[]
* You can use {gcp-short} Hyperdisk storage with {VirtProductName} on {product-title} on {gcp-short}. {gcp-short} Hyperdisk storage provides high performance and flexibility for VM workloads. For more information about using Hyperdisk storage, see link:https://access.redhat.com/articles/7139046[Storage configuration for OpenShift Virtualization 4.21.x on Google Cloud].
* In {product-title} on {gcp-short}, you must ensure your StorageClass uses the GCP PD CSI driver or {gcp-short} Filestore CSI driver.
+
[NOTE]
====
For more information about configuring {VirtProductName} on {product-title} on {gcp-short}, see link:https://access.redhat.com/articles/7139682[OpenShift Virtualization on Google Cloud: Known Issues and Limitations].
====
endif::openshift-dedicated[]
+
ifdef::openshift-rosa,openshift-rosa-hcp[]
[IMPORTANT]
====

> {aws-short} bare metal, {product-rosa} classic architecture clusters might have different supported storage solutions. Ensure that you confirm support with your storage vendor.

Reviewer: ```suggestion
{aws-short} bare metal, {product-rosa}, and {product-rosa} classic architecture clusters might have different supported storage solutions. Ensure that you confirm support with your storage vendor.
```

My understanding is that {product-rosa} and {product-rosa} classic are two different things, so we should keep both instances here (though I can see why it looked like an error).

ifdef::openshift-rosa,openshift-rosa-hcp[]
[IMPORTANT]
====

> {aws-short} bare metal and ROSA clusters might have different supported storage solutions. Ensure that you confirm support with your storage vendor.

Reviewer 2: ```suggestion

```
This seems like a duplicate of the line below

> {aws-short} bare metal, Red Hat OpenShift Service on {aws-short}, and Red Hat OpenShift Service on {aws-short} classic architecture clusters might have different supported storage solutions. Ensure that you confirm support with your storage vendor.

Reviewer 2: ```suggestion
{aws-short} bare metal and {product-rosa} classic architecture clusters might have different supported storage solutions. Ensure that you confirm support with your storage vendor.
```

Author: @sjhala-ccs Done, thank you. Please merge.
